Sources/AirRobeWidget/Resources/AirRobeStrings.swift

```diff
@@ -91,4 +95,8 @@ enum AirRobeStrings {
 
     // MARK: - Delimiter
     static let delimiter: String.Element = "/"
+
+    // MARK: - Target Split Test Variants
+    static let airrobeEnabled = "airrobe-enabled"
```

Comment:
Hey mate, actually, these strings, "airrobe-enabled" and "airrobe-disabled" should *not* be considered special. Think about the target_split_test_variant property as a label that must simply be sent with telemetry events. Any string value could be present here.

Whether or not airrobe is shown depends only on the "disabled" property in the variant.

Reply:
You meant `enabled` property right?

Reply:
Ahh, right. In web we actually flip the config around to use `disabled`, instead of `enabled` - are you ok if we make that update before merging? I'll update the connector!

Reply:
No problem. 👍

Reply:
Let's do it @michaeldawson

Reply:
Cheers mate! Here's the reference PR with changes :) https://github.com/airrobe/connector/pull/384